Tests/MeiliSearchIntegrationTests/Utils.swift

```diff
@@ -20,7 +20,7 @@ private let movies: [Movie] = [
 public let TESTS_TIME_OUT = 10.0
 
 public func currentHost() -> String {
-  ProcessInfo.processInfo.environment["MEILISEARCH_HOST"] ?? "http://localhost:7700"
+  ProcessInfo.processInfo.environment["MEILISEARCH_URL"] ?? "http://localhost:7700"
```

Comment:
I can't comment in other places, but can you also change the name of the `host` param? It now should also be `url`.

example:
`client = try MeiliSearch(host: "http://localhost:7700", apiKey: "masterKey")` become `client = try MeiliSearch(url: "http://localhost:7700", apiKey: "masterKey")`

also, in the `code-samples.meilisearch.yml` and over the README :)

Reply:
Better thinking is not the best way to forward with that. We should deprecate the current param first. Then in another PR and another release version, remove it.

Reply:
So, can you "deprecate" the `host` from `MeiliSearch` param to merge this PR?

Reply:
Better thinking again, we can do that later. Thanks!